### ISSUE TITLE
Shapely 1.6.x is not supported by MapProxy 1.10.*

### ISSUE THREAD

**Issue by ddegasperi** [mapproxy/mapproxy#334]
I've receiving the following error message, when I've try to load coverages:

```

Traceback (most recent call last):
  File "/usr/local/mapproxy/bin/mapproxy-seed", line 11, in <module>
    sys.exit(main())
  File "/usr/local/mapproxy/lib/python2.7/site-packages/mapproxy/seed/script.py", line 360, in main
    return SeedScript()()
  File "/usr/local/mapproxy/lib/python2.7/site-packages/mapproxy/seed/script.py", line 218, in __call__
    seed_tasks = seed_conf.seeds(seed_names)
  File "/usr/local/mapproxy/lib/python2.7/site-packages/mapproxy/seed/config.py", line 201, in seeds
    seed_conf = SeedConfiguration(seed_name, seed_conf, self)
  File "/usr/local/mapproxy/lib/python2.7/site-packages/mapproxy/seed/config.py", line 282, in __init__
    ConfigurationBase.__init__(self, name, conf, seeding_conf)
  File "/usr/local/mapproxy/lib/python2.7/site-packages/mapproxy/seed/config.py", line 228, in __init__
    self.coverage = self._coverages()
  File "/usr/local/mapproxy/lib/python2.7/site-packages/mapproxy/seed/config.py", line 237, in _coverages
    coverages = [self.seeding_conf.coverage(c) for c in self.conf.get('coverages', {})]
  File "/usr/local/mapproxy/lib/python2.7/site-packages/mapproxy/util/py.py", line 79, in wrapper
    cache[key] = func(self, *args, **kwargs)
  File "/usr/local/mapproxy/lib/python2.7/site-packages/mapproxy/seed/config.py", line 153, in coverage
    coverage = load_coverage(coverage_conf)
  File "/usr/local/mapproxy/lib/python2.7/site-packages/mapproxy/config/coverage.py", line 81, in load_coverage
    require_geom_support()
  File "/usr/local/mapproxy/lib/python2.7/site-packages/mapproxy/util/geom.py", line 51, in require_geom_support
    raise ImportError('Shapely required for geometry support')
ImportError: Shapely required for geometry support

```

## Context

I've installed MapProxy 1.10.4 and the latest version of the Shapely library (1.6.2). When I use the previous version Shapely 1.5.17, then it works as expected.

## Expected Behavior

If MapProxy 1.10.x does not support the newer version of the Shapely library, I've expected a dependency error, when I run pip install Shapely or it automatically installs the max supported version of the library

## Actual Behavior

If I install Shapely via pip install shapely, it installs automatically the latest version 1.6.2

**Comment by olt** [comment]
MapProxy does work with Shapely 1.6.2. Can you reinstall Shapely 1.6.2 copy the output?

**Comment by olt** [comment]
I think this is related to https://github.com/mapproxy/mapproxy/commit/928b185a9737bb9472920a3bedd0bc25ed80a80b
So this is fixed in master and will be fixed in 1.11.0.

**Comment by fdetsch** [comment]
Same here when working with GeoJSON or ESRI Shapefile coverages: 

```
Traceback (most recent call last):
  File "c:\users\fdetsch\anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\fdetsch\anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\fdetsch\Anaconda3\Scripts\mapproxy-util.exe\__main__.py", line 9, in <module>
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\script\util.py", line 387, in main
    commands[command]['func'](args)
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\script\util.py", line 85, in serve_develop_command
    app = make_wsgi_app(mapproxy_conf, debug=options.debug)
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\wsgiapp.py", line 110, in make_wsgi_app
    services = conf.configured_services()
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\config\loader.py", line 211, in configured_services
    return self.services.services()
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\config\loader.py", line 1770, in services
    new_services = creator(service_conf or {})
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\config\loader.py", line 1921, in demo_service
    lyr = layer_conf.wms_layer()
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\util\py.py", line 78, in wrapper
    cache[key] = func(self, *args, **kwargs)
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\config\loader.py", line 1619, in wms_layer
    map_layer = self.context.caches[source_name].map_layer()
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\util\py.py", line 78, in wrapper
    cache[key] = func(self, *args, **kwargs)
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\config\loader.py", line 1536, in map_layer
    for grid, extent, tile_manager in self.caches():
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\util\py.py", line 78, in wrapper
    cache[key] = func(self, *args, **kwargs)
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\config\loader.py", line 1445, in caches
    request_format=request_format,
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\config\loader.py", line 1339, in _sources_for_grid
    source = source_conf.source({'format': request_format})
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\config\loader.py", line 949, in source
    coverage = self.coverage()
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\util\py.py", line 78, in wrapper
    cache[key] = func(self, *args, **kwargs)
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\config\loader.py", line 545, in coverage
    return load_coverage(self.conf['coverage'])
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\config\coverage.py", line 81, in load_coverage
    require_geom_support()
  File "c:\users\fdetsch\anaconda3\lib\site-packages\mapproxy\util\geom.py", line 55, in require_geom_support
    raise ImportError('Shapely required for geometry support')
ImportError: Shapely required for geometry support
```

MapProxy 1.11.0 is running in a conda environment: 

```
(MapProxy) C:\Users\fdetsch\mymapproxy>mapproxy-util --version
MapProxy 1.11.0
```

Shapely has been installed using pip (as suggested [here](https://mapproxy.org/docs/nightly/coverages.html)) and should hence be available: 

```
(MapProxy) C:\Users\fdetsch\mymapproxy>pip show Shapely
Name: Shapely
Version: 1.6.4.post1
Summary: Geometric objects, predicates, and operations
Home-page: https://github.com/Toblerity/Shapely
Author: Sean Gillies
Author-email: sean.gillies@gmail.com
License: BSD
Location: c:\users\fdetsch\anaconda3\envs\mapproxy\lib\site-packages
Requires:
```
